### PR TITLE
Update metadata.json for 1.2.34 release

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$subject

### CODE REVIEW

File: _data/stable-latest/metadata.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
 {
-	"version":"1.2.33",
-	"release-date":"2022-10-03",
-	"windows-installer":"ballerina-windows-installer-x64-1.2.33.msi",
+	"version":"1.2.34",
+	"release-date":"2022-11-08",
```
Comment (NipunaRanasinghe) [inline _data/stable-latest/metadata.json:3]:
```suggestion
	"release-date":"2022-11-09",
```

File: _data/stable-latest/metadata.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 	"windows-installer-size":"145mb",
-	"linux-installer":"ballerina-linux-installer-x64-1.2.33.deb",
+	"linux-installer":"ballerina-linux-installer-x64-1.2.34.deb",
 	"linux-installer-size":"139mb",
-	"macos-installer":"ballerina-macos-installer-x64-1.2.33.pkg",
+	"macos-installer":"ballerina-macos-installer-x64-1.2.34.pkg",
 	"macos-installer-size":"161mb",
-	"rpm-installer":"ballerina-linux-installer-x64-1.2.33.rpm",
+	"rpm-installer":"ballerina-linux-installer-x64-1.2.34.rpm",
 	"rpm-installer-size":"164mb",
```
Comment (NipunaRanasinghe) [inline _data/stable-latest/metadata.json:11]: Have we checked the new installer sizes?

Comment (dilanSachi) [reply _data/stable-latest/metadata.json:11]: I copied the values from the https://dist-dev.ballerina.io/downloads/1.2.34/metadata.json. Ill check it and confirm.

File: _data/stable-latest/metadata.json (hunk 1 above)
Comment (dilanSachi) [reply _data/stable-latest/metadata.json]: Updated

File: _data/stable-latest/metadata.json (hunk 2 above)
Comment (dilanSachi) [reply _data/stable-latest/metadata.json]: Updated with the real file sizes

**Comment by dilanSachi** [comment]
Waiting until the go ahead is given by MGW

**Comment by dilanSachi** [comment]
> Waiting until the go ahead is given by MGW

Has been tested and got the +1

**Comment by dilanSachi** [comment]
> Waiting until the go ahead is given by MGW

+1 is received.

**Review by keizer619 — APPROVED** [review]

**Review by praneesha — APPROVED** [review]

**Pull request merged**